### PR TITLE
fail at test

### PR DESCRIPTION
**Pull request by Krzychu92**
wywala błąd przy npm test

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,51 @@
...
+            <a class="nav__link active" href="#apple">Apple</a>
+          </li>
+          <li>
+            <a class="nav__link active" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__list">
+            <a class="nav__link active" href="#Smartphones">Smartphones</a>
+          </li>
+          <li class="nav__list">
+            <a class="nav__link active" href="#Laptops_&_Computers">Laptops & Computers</a>
```
Comment (mykhalenych) [inline src/index.html:32]: missed data-qa="hover"

File: src/index.html [hunk 2]
```diff
@@ -1,14 +1,51 @@
...
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#home"><img src="./images/logo.png" alt="logo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a class="nav__link active" href="#apple">Apple</a>
+          </li>
+          <li>
+            <a class="nav__link active" href="#samsung">Samsung</a>
```
Comment (mykhalenych) [inline src/index.html:26]: why all your links are active? only first must be active

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,52 @@
...
+  color: black;
+  cursor: pointer;
+  padding: 20px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.active:focus {
```
Comment (mykhalenych) [inline src/style.css:41]: change to ::after with position absolute

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,52 @@
...
+
+.nav__link {
+  font-weight: bold;
+}
+
+nav li {
+  display: inline-block;
+}
+
+nav a {
```
Comment (mykhalenych) [inline src/style.css:27]: you already have classname for a tag, use class instead of tag name

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,52 @@
...
+  text-decoration: none;
+  font-family: "Roboto", sans-serif;
```
Comment (mykhalenych) [inline src/style.css:29]: move to html selector

**Review by mykhalenych — CHANGES_REQUESTED** [review]
Don't forget add links to description, check comments and design
https://www.figma.com/file/1sog2rmfyCjnVxkeZ3ptnc/MOYO-%2F-Header?type=design&node-id=0-1&mode=design&t=e5Ni7UCJlaNNGVHp-0

File: src/index.html (hunk 1 above)
Comment (Krzychu92) [reply src/index.html:32]: that data-qa="hover" is for li or a?

**Pull request closed without merging**